packages/management-console/src/components/Organisms/DomainExplorerColumnPicker/DomainExplorerColumnPicker.tsx

```diff
@@ -0,0 +1,239 @@
+  columnPickerType,
+  setColumnFilters,
+  setTableLoading,
+  getQueryTypes,
+  setDisplayTable
+}) => {
+  const [selected, setSelected] = useState([]);
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [parameters, setParameters] = useState([]);
+
+  const nullTypes = [null, 'String', 'Boolean', 'Int', 'DateTime'];
+  const onSelect = (event, selection) => {
+    if (selected.includes(selection)) {
+      setSelected(prevState => prevState.filter(item => item !== selection));
+      const innerText = event.nativeEvent.target.nextSibling.innerText;
+      const rest = filterColumnSelection(event, innerText);
+      setParameters(prevState =>
+        prevState.filter(obj => {
+          if (!_.isEqual(obj, rest)) {
+            return obj;
+          }
+        })
+      );
+    } else {
+      setSelected(prevState => [...prevState, selection]);
+      const innerText = event.nativeEvent.target.nextSibling.innerText;
+      const rest = filterColumnSelection(event, innerText);
+      setParameters(prevState => [...prevState, rest]);
+    }
+  };
+
+  const filterColumnSelection = (event, selection) => {
+    const parent = event.nativeEvent.target.parentElement.parentElement.getAttribute(
+      'aria-labelledby'
+    );
+    let res = {};
+    const tempParents = parent.split('-');
+
+    for (let i = tempParents.length - 1; i >= 0; i--) {
+      if (i === tempParents.length - 1) {
+        res = { [tempParents[i]]: [selection] }; // assign the value
+      } else {
+        res = { [tempParents[i]]: [res] }; // put the prev object
+      }
+    }
+    return res;
+  };
+  const onToggle = _isExpanded => {
+    setIsExpanded(_isExpanded);
+  };
+
+  const getPicker = useGetPickerQuery({
+    variables: { columnPickerType }
+  });
+
+  async function generateQuery() {
+    try {
+      await axios
+        .post(
+          'http://localhost:4000/graphql',
+          query({
+            operation: columnPickerType,
+            fields: parameters
+          })
+        )
+        .then(response => {
+          setTableLoading(false);
+          setColumnFilters(response.data.data);
+          setDisplayTable(true);
+          return response;
+        });
+    } catch (error) {
+      return error;
+    }
+  }
+
+  const data = [];
+  !getPicker.loading &&
+    getPicker.data.__type &&
+    getPicker.data.__type.fields.filter(i => {
+      if (i.type.kind !== 'SCALAR') {
+        return data.push(i);
+      }
+    });
+  const fetchSchema = option => {
+    return (
+      !getQueryTypes.loading &&
+      getQueryTypes.data.__schema &&
+      getQueryTypes.data.__schema.queryType.find(item => {
+        if (item.name === option.type.name) {
+          return item;
+        }
+      })
+    );
+  };
+
+  let b;
+  let finalResult: any = [];
+  let a: any;
+
+  const childSelectionItems = (_data, title, ...attr) => {
+    let nestedTitles = '';
+    b = _data.map(group => {
+      const label = title + '>' + attr.join();
```

Comment:
```suggestion
      const label = title + ' > ' + attr.join();
```
just adding a extra space here, before and after the string.

Reply:
done